src/shared/components/ncTable/sections/CustomTable.vue

```diff
@@ -464,7 +464,7 @@ export default {
 
 	thead {
 		position: sticky;
-		top: 118px;
+		top: 108px;
```

Comment:
not sure about this one

Reply:
I'd say fine as a quick fix, Ideally we could define a var with the height of the table title element and reuse that then